correction/replacement.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+
+from .types import ParseTree, QueryFunction, ReplaceFunction, Path
+
+
+class Replacement:
+    '''Represents a single replacement.
+    A replacement is a combination of search/match queries,
+    and a replacing value. It operates on Lassy XML of
+    one sentence/utterance.
+    '''
```

Comment:
Perhaps you can clarify:

- the replacement object is not a replacing _value_ but a function that executes a replacement.
- multiple replacements can be included (with order of execution)
- While this is implied in the types, perhaps you can also add that the replacement function must work in-place, rather than as a pure function.
- Add that the combination of search/match queries will be applied with AND/intersect logic